packages/components/psammead-media-player/src/Guidance/index.jsx

```diff
@@ -12,16 +12,32 @@ const GUIDANCE_BACKGROUND = 'rgba(34, 34, 34, 0.75)';
 const GuidanceWrapper = styled.div`
   ${({ service }) => getSansRegular(service)}
   ${GEL_LONG_PRIMER};
-
   width: 100%;
   height: 100%;
   position: absolute;
-  background-color: ${GUIDANCE_BACKGROUND};
   border: 0.0625rem solid transparent;
   color: ${C_WHITE};
+  ${({ guidanceMessage }) =>
+    guidanceMessage ? `background-color: ${GUIDANCE_BACKGROUND};` : ``}
+
   @media screen and (-ms-high-contrast: active) {
```

Comment:
This media query is rendered with no contents when there is no guidance message. 
I'd suggest wrapping all of this section in the guidanceMessage check. e.g.
```
  ${({ guidanceMessage }) =>
    guidanceMessage
      ? `
        background-color: ${GUIDANCE_BACKGROUND};
        @media screen and (-ms-high-contrast: active) {
          background-color: transparent;
        }`
      : ``
  }
```

Reply:
done